poet/sawtooth_poet/validator_registry/processor/handler.py

```diff
@@ -121,11 +121,9 @@ class ValidatorRegistryTransactionHandler(object):
         '5Jz5Kaiy3kCiHE537uXcQnJuiNJshf2bZZn43CrALMGoCd3zRuo'
 
     def __init__(self):
-        self._report_private_key = \
-            signing.encode_privkey(
-                signing.decode_privkey(self.__REPORT_PRIVATE_KEY_WIF, 'wif'),
-                'hex')
-
+        # Since signing works with WIF-encoded private keys, we don't have to
```

Comment:
It feels like most of these comments are explanations of the change (commit entry type stuff), rather than a description of what the code is doing. I'd recommend removing all the comments about not having to encode/decode keys.